packages/core/admin/webpack.config.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
 const path = require('path');
+const fse = require('fs-extra');
 const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
I think you can remove `terser-webpack-plugin` as dependency now from `package.json`?